Python/Product/Debugger/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -1482,8 +1483,8 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
             return null;
         }
 
-        internal async void RefreshThreadFrames(long threadId) {
-            await _process.GetThreadFramesAsync(threadId);
+        internal async System.Threading.Tasks.Task RefreshThreadFrames(long threadId) {
+            await _process.GetThreadFramesAsync(threadId).ConfigureAwait(false);
```

Comment:
`ConfigureAwait(false)` here has no impact as the outer caller will need to do it as well.

In this case, just do `return _process.GetThreadFramesAsync(threadId)` and remove the `async` from the declaration.